apps/src/aichat/types.ts

```diff
@@ -54,6 +63,12 @@ export interface AiCustomizations {
   modelCardInfo: ModelCardInfo;
 }
 
+export interface AiCustomizationsForBackend {
```

Comment:
Adding bc there is no need to send `modelCardInfo` to backend.

Reply:
Nice! Can AiCustomizations extend this then?

Reply:
We could also use TS's `Omit` here and just simplify this to 
```
export type AiCustomizationsForBackend = Omit<AiCustomizations, 'modelCardInfo'>
```
Also, thoughts on naming this something like `AichatParameters/Settings` or something?

Reply:
Cool! I'll use `Omit` and rename as `AichatParameters`.